docs/book/user-guide/advanced-guide/custom-stack-solutions/implement-a-custom-stack-component.md

```diff
@@ -4,21 +4,39 @@ description: How to write a custom stack component flavor
 
 # Implement a custom stack component
```

Comment:
See here it's 'implement'. So maybe in the TOC it can be that as well?

Reply:
H1 headers aren't shown so doesn't matter but yeah I also like implement best 👍